include/cuda/std/detail/libcxx/include/span

```diff
@@ -163,81 +163,82 @@ template <class _Tp, size_t _Sz>
 struct __is_std_array_impl<array<_Tp, _Sz>> : public true_type {};
 
 template <class _Tp>
-struct __is_std_array : public __is_std_array_impl<remove_cv_t<_Tp>> {};
+struct __is_std_array : public __is_std_array_impl<typename remove_cv<_Tp>::type> {};
 
+#if _LIBCUDACXX_STD_VER > 14
```

Comment:
Why is this C++17+ only?

Reply:
The container overloads that use this specialization require `std::data` and `std::size` which are both only available in C++17

We can move them up to C++14 of course but I wanted to discuss this and forgot to write the comment

Reply:
Oh that's very silly. I think we should backport them - they don't rely on any core language things that are c++17, right?

Reply:
Done